airbyte-webapp/src/components/connection/CatalogTree/next/NextTable/NextTable.module.scss

```diff
@@ -0,0 +1,73 @@
+@use "scss/colors";
+
+// ------- <table/> --------
+.table {
+  border-spacing: 0;
+  width: 100%;
+  max-width: 100%;
+  border-radius: 10px;
+}
+
+.thead {
+  position: sticky;
+  top: 0;
+}
+
+// --------- <th/> ---------
+.th {
+  background: colors.$grey-50;
+  color: colors.$grey;
+  text-transform: uppercase;
+  border-bottom: none;
+  text-align: left;
+  font-size: 10px;
+  line-height: 12px;
+  font-weight: 600;
+
+  &:first-child {
+    padding-left: 45px;
+    border-radius: 10px 0 0;
+  }
+
+  &:last-child {
+    padding-left: 15px;
+    border-radius: 0 10px 0 0;
+  }
+}
+
+// --------- <tr/> ---------
+.tr {
+  background: colors.$white;
+  cursor: auto;
+
+  &:hover {
+    background-color: colors.$grey-50;
+  }
+}
+
+// --------- <td/> ---------
+
+.td {
+  padding: 16px 13px;
+  font-size: 12px;
+  line-height: 15px;
+  font-weight: normal;
+  color: colors.$dark-blue;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  border-bottom: 1px solid colors.$grey-100;
+  width: auto;
+
+  tr:last-child > & {
+    border-bottom: none;
+
+    &:first-child {
+      border-radius: 0 0 0 10px;
```

Comment:
The same for this **$border-radius-md: 10px;**

Reply:
fixed